driftbase/api/clients.py

```diff
@@ -235,6 +237,7 @@ def post(self, args):
 
         current_app.extensions["messagebus"].publish_message("client", message_data)
 
+        richpresence.set_online_status(player_id, True)
```

Comment:
You want to ensure this never throws, or wrap it and ignore failures, since this is a side effect and not a requirement for the success of POST /clients